File: android/ffmpeg-kit-android-lib/src/main/java/com/lgcns/ffmpegkit/Signal.java
package com.lgcns.ffmpegkit;

/**
 * <p>Lists signals handled by FFmpegKit library.
 */
public enum Signal {

    SIGINT(2),
    SIGQUIT(3),
    SIGPIPE(13),
    SIGTERM(15),
    SIGXCPU(24);

    private final int value;

    Signal(int value) {
        this.value = value;
    }

    public int getValue() {
        return value;
    }

}

File: android/ffmpeg-kit-android-lib/src/main/java/com/lgcns/ffmpegkit/Signal.kt
package com.lgcns.ffmpegkit

/**
 *
 * Lists signals handled by FFmpegKit library.
 */
enum class Signal(@JvmField val value: Int) {
    SIGINT(2), SIGQUIT(3), SIGPIPE(13), SIGTERM(15), SIGXCPU(24)

}